### PR TITLE
Fix `Download`  error when no file is found

### PR DESCRIPTION
**Pull request by AllardQuek**
- Throw an exception and display popup message to user alerting them download failed

### CODE REVIEW

File: src/main/java/seedu/programmer/commons/util/JsonUtil.java [hunk 1]
```diff
@@ -186,15 +187,15 @@ public static void writeJsonToCsv(JSONArray jsonData, File destinationFile) {
      * @param filePath Path of file to get JSON data from.
      * @return JSONArray of student's data.
```
Comment (shermannws) [inline src/main/java/seedu/programmer/commons/util/JsonUtil.java:188]: I think can all the Exception thrown in the javadoc too

**Review by shermannws — CHANGES_REQUESTED** [review]
Propose to amend Javadoc

File: src/main/java/seedu/programmer/commons/util/JsonUtil.java (hunk 1 above)
Comment (AllardQuek) [reply src/main/java/seedu/programmer/commons/util/JsonUtil.java:188]: Good catch!

**Review by shermannws — APPROVED** [review]
LGTM!

**Pull request merged**